ieee/.htaccess

```
@@ -27,6 +27,12 @@ RewriteRule ^ieee-2791-schema/(.+)$ https://opensource.ieee.org/2791-object/ieee
 ## link to only ^ieee-2791-schema or ^ieee-2791-schema/
 RewriteRule ^ieee-2791-schema(/)?$ https://opensource.ieee.org/2791-object/ieee-2791-schema/ [R=302,L]
 
+##
+## ^ieee-1451.0-schema/
+##
+
+RewriteRule ^ieee-1451.0-schema/((examples/)?\w+).xsd$ https://opensource.ieee.org/1451/1451.0-schema/-/blob/main/normative/$1.xsd [R=302,L]
```

I made an error in this simplification.  The original PR, https://github.com/perma-id/w3id.org/pull/3262, maps the examples from xsd to xml.  Two rules will address the issue:
```suggestion
RewriteRule ^ieee-1451.0-schema/(\w+).xsd$ https://opensource.ieee.org/1451/1451.0-schema/-/blob/main/normative/$1.xsd [R=302,L]
RewriteRule ^ieee-1451.0-schema/examples/(\w+).xsd$ https://opensource.ieee.org/1451/1451.0-schema/-/blob/main/normative/$1.xml [R=302,L]
```
- Should those be using `raw` instead of bringing up the `blob` UI?  I don't know the use case here to determine what you'd want.
- Should the examples be mapping `xsd` to `xml` or should that be `xml` to `xml`?

---

Thank you for your help. XML -> XML and XSD -> XSD. I will update accordingly